org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/debugger/server/rmi/XWollokObjectDebugValue.xtend

```diff
@@ -41,6 +41,7 @@ class XWollokObjectDebugValue extends XDebugValue {
 	}
 	
 	def isList() { typeName == LIST }
+	def isDictionary() { typeName == DICTIONARY }
```

Comment:
Eventually we will add a custom view in the objects diagram to show dictionaries, maybe like a 2 columns table, or something (needs to be more clever, to decide if it will put the value and key object within the table or just an arrow to an object outside the table.
#526 new issue